### ISSUE TITLE
ScanAPI pod keeps restarting

### ISSUE THREAD

**Issue by czunker** [mondoohq/mondoo-operator#813]
**Describe the bug**
The ScanAPI pod is stuck in a `CrashLoopBackOff`. In the logs, you can see a message like
```
FTL could not enable scan queue error="unable to create queue segment in /tmp/cnspec-queue/disk-queue: unable to load queue segment in /tmp/cnspec-queue/disk-queue: segment file /tmp/cnspec-queue/disk-queue/0000000000047.dque is corrupted: error reading gob data from file: EOF"
```

**To Reproduce**
We do not have a way to reproduce this. It happens from time to time. We saw this in different environments already.

**Expected behavior**
The ScanAPI should start.

**Additional context**
Perhaps https://github.com/joncrlsn/dque/issues/33 helps to reproduce this.

**Comment by czunker** [comment]
The current workaround is to delete the Pod and wait for the Deplyoment to start a new.

**Comment by imilchev** [comment]
Fixed with mondoohq/cnspec#951
